airmozilla/main/static/main/js/eventstatus.js

```diff
@@ -0,0 +1,22 @@
+$(function () {
+    var TEN_SECONDS = 10000;
```

Comment:
Ha! I just made my next point for myself. I accidentally typo'ed 100000. One 0 too many. It's easier to read `10 * 1000` and it's a common convention when specifying milliseconds. Already calculating that `10 * 1000` equals `10000` is not a good optimization.